Makefile

```diff
@@ -7,7 +7,16 @@ INSTANCE = catalogue
 
 .PHONY: default copy test
 
-default: test
+default: build
+
+pre:
+	go get -u github.com/FiloSottile/gvt
+
+deps: pre
+	gvt restore
+
+rm-deps:
+	rm -rf vendor
```

Comment:
Only remove the directories, gvt is storing the version of each dependency in the vendor/manifest file.
Something like this would work:
```find vendor -depth 1 -type d -exec rm -r "{}" +```